go/libkb/util.go

```diff
@@ -783,3 +785,26 @@ func IsNoSpaceOnDeviceError(err error) bool {
 
 	return false
 }
+
+func ShredFile(filename string) error {
+	stat, err := os.Stat(filename)
+	if err != nil {
+		return err
+	}
+	if stat.IsDir() {
+		return errors.New("cannot shread a directory")
+	}
+	size := int(stat.Size())
+
+	for i := 0; i < 3; i++ {
+		noise, err := RandBytes(size)
+		if err != nil {
+			return err
+		}
+		if err := ioutil.WriteFile(filename, noise, stat.Mode().Perm()); err != nil {
+			return err
```

Comment:
sorry to nitpick, but if this fails, i think we still want to delete the file... And log it with a big warning?

Reply:
Good point...I'll need to contextify SecretStoreFile to get a log message in here.  But I pushed a change that will always call os.Remove.

Reply:
Cool, but looks like you're calling `Remove` twice on the way out now?

Reply:
It doesn't matter if the second one fails...